types/index.d.ts

```diff
@@ -9,18 +10,41 @@ interface Signature {
   sig: string
 }
 
+// https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository
 interface Repository {
   directory?: string
   type?: string
   url: string
 }
 
+// https://docs.npmjs.com/cli/v10/configuring-npm/package-json#funding
+interface Funding {
+  type: string
+  url: string
+}
+
+// https://docs.npmjs.com/cli/v10/configuring-npm/package-json#overrides
+interface Override {
```

Comment:
@broofa Overrides here can be recursive according to the docs:

> Keys can be nested to any arbitrary length. To override foo only when it's a child of bar and only when bar is a child of baz:

```
{
  "overrides": {
    "baz": {
      "bar": {
        "foo": "1.0.0"
      }
    }
  }
}
```

```suggestion
interface Override {
  [moduleName: string]: string | Override
}
```

Reply:
Fixed.  Also renamed `Override` -> `Overrides` for consistency with `package#dependencies` property.